as2/6.3/modules/ROOT/pages/as2-connector-example-multiple-partners.adoc

```diff
@@ -0,0 +1,149 @@
+= Sending MDNs to Multiple Partners - Mule 4
+:page-aliases: connectors::as2/as2-connector-multiple-partners.adoc
+
+The following example shows how to send Message Disposition Notifications (MDNs) to multiple partners. This example contains two flows:
+
+* The first flow configures an AS2 listener endpoint to receive inbound AS2 messages.
++
+* The second flow receives message payloads and AS2 sender and receiver information via HTTP. It then uses the *Send with Sync MDN* operation to send outbound AS2 messages to a target AS2 endpoint.
+
+This example is similar to the xref:as2-connector-example-receive-send.adoc[Receiving AS2 Messages and Sending Back MDNs] example, except that for this example:
+
+* The partner information must already exist in the configured keystore.
+* The partner information must be configured as follows in the *Partners* tab of the global element:
+** The *Listener Mode* field must be set to *Shared mode* and details are specified for the additional partner.
+** The *Partner config* section of this tab, which is highlighted in the following screenshot, must be left blank or the connector returns errors:
```

Comment:
Is there any way to avoid passive voice here? The first sentence is tricky, but for the others it could be like:
 
* You must configure partner information as follows in...
** Set the *Listener Mode* field to *Shared mode* and specify details for...
** Leave *Partner config* section of this tab blank or ...

Reply:
In this case, it's a choice between removing the passive tense from the second step and its substeps or having parallel bullets.